assets/src/components/unsaved-changes-warning/index.js

```diff
@@ -12,7 +12,7 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { User } from '../../onboarding-wizard/components/user-context-provider';
+import { User } from '../../components/user-context-provider';
```

Comment:
Import can be shortened.

```suggestion
import { User } from '../user-context-provider';
```